latest/tests/image_suite.json

```diff
@@ -557,80 +465,6 @@
       },
       "valid": false
     },
-    {
-      "formerly": "invalid/invalid_axes_count.json",
```

Comment:
Instead of deleting these, it might be worth just flipping valid=true?

Reply:
Next steps here?

Reply:
i decided the tests were not meaningful as positive examples